### PR TITLE
Improve configurable learn page

### LINKED ISSUE #8862: Improve configurable learn page to include env variable based configuration
## Description
$subject

## Related website/documentation area
Area/LearnPages

## Describe the problem(s)
https://github.com/ballerina-platform/ballerina-lang/issues/42083

### PR DESCRIPTION
**Pull request by HindujaB**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md [hunk 1]
```diff
@@ -10,13 +10,41 @@ intro: You can provide values for configurable variables through multiple method
...
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| boolean        | <code>configurable boolean isAdmin = ?; </code>                                                                                                                                                               | `$ set BAL_CONFIG_VAR_ISADMIN=true` <br> or <br> `$ set BAL_CONFIG_VAR_ISADMIN=1` | `$ export BAL_CONFIG_VAR_ISADMIN=true` <br> or <br> `$ export BAL_CONFIG_VAR_ISADMIN=1` |
```
Comment (sm1990) [inline swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md:39]: Is there a reason to use <code> tags here? You should be able to use back ticks.

Comment (HindujaB) [reply swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md:39]: Some parts contain multiline code that includes Balerina record definitions. In earlier versions, it didn't render properly when we used backticks.

**Comment by sm1990** [comment]
LGTM

File: swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md [hunk 2]
```diff
@@ -93,7 +123,7 @@ $ set BAL_CONFIG_FILES=<path-to-info.toml>;<path-to-Config.toml>
 $ export BAL_CONFIG_FILES=<path-to-info.toml>:<path-to-Config.toml>
 ```
 
-## Provide via environment variables
+### Provide via environment variables with TOML content
```
Comment (anupama-pathirage) [inline swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md:126]: This is under the title with `Provide using TOML content`, Then why do we need to add `with TOML content` to this sub title. IMO this is not necessary and we don't have that part in previous sub title `Provide via configuration files` as well.

File: swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md [hunk 3]
```diff
@@ -8,15 +8,43 @@ active: provide-values-to-configurable-variables
 intro: You can provide values for configurable variables through multiple methods, as described below.
 ---
 
->**Note:** If the configuration values are given in multiple ways, they will be overridden in the following precedence order when retrieving them.
+>**Note:** If the configuration values are given in multiple ways, they will be overridden in the following decreasing order of precedence when retrieving them.
```
Comment (warunalakshitha) [inline swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md:11]: This seems unclear. how about 
**Note:** If the configuration values are given in multiple ways, they will be overridden in the following order when retrieving them.

**Review by warunalakshitha — APPROVED** [review]

**Review by warunalakshitha — APPROVED** [review]

**Pull request merged**